src/ui/api/search.go

```diff
@@ -76,17 +69,19 @@ func (s *SearchAPI) Get() {
 			continue
 		}
 
-		if userID != dao.NonExistUserID {
-			roles, err := dao.GetUserProjectRoles(userID, p.ProjectID)
+		if isAuthenticated {
```

Comment:
The codes not only populate the public property of project, but also set the role which user has to the project.